### PR TITLE
 [bitnami/grafana-mimir] removing quotes from gw secrets #29214

### PR DESCRIPTION
**Pull request by netikras**
### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by juan131** [comment]
Hi @netikras 

Did you find any issue with the quotes? They shouldn't be problematic given they're YAML strings.

**Comment by netikras** [comment]
> Hi @netikras
> 
> Did you find any issue with the quotes? They shouldn't be problematic given they're YAML strings.

Hi!

Yes, the quotation is doubled, effectively making helm to escape inner quotes.

see: https://github.com/bitnami/charts/issues/29214

File: bitnami/grafana-mimir/templates/gateway/secret.yaml [hunk 1]
```diff
@@ -20,6 +20,6 @@ metadata:
 type: Opaque
 data:
   {{- $password := (include "common.secrets.passwords.manage" (dict "secret" (include "grafana-mimir.gateway.fullname" .) "key" "password" "providedValues" (list "gateway.auth.password") "context" $)) }}
-  password: {{ $password | quote }}
-  htpasswd: {{ htpasswd .Values.gateway.auth.username (b64dec $password) | b64enc | quote }}
+  password: {{ $password }}
+  htpasswd: {{ htpasswd .Values.gateway.auth.username (b64dec $password) | b64enc }}
```
[the remark this answers is not in the document]
Comment (netikras) [reply bitnami/grafana-mimir/templates/gateway/secret.yaml:24]: @juan131   Do we expect base64-encoded values to clash with yaml syntax?

[the remark this answers is not in the document]
Comment (netikras) [reply bitnami/grafana-mimir/templates/gateway/secret.yaml]: done

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**